tests/repositories/firestore/test_rate.py

```
@@ -0,0 +1,286 @@
+
+        token = self.gen_token_employee(role=Role.AGENT, client_id=str(self.client_id), assigned=True)
+        headers = {'X-Apigateway-Api-Userinfo': self.encode_token(token)}
+
+        resp = self.test_client.get('/api/v1/invoice', headers=headers)
+
+        self.assertEqual(resp.status_code, 403)
+        self.assertIn("Forbidden", resp.get_json()["message"])
+
+    @patch("google.cloud.firestore_v1.client.Client.__init__", return_value=None)
+    @patch("google.cloud.firestore_v1.client.Client.collection")
+    def test_get_invoice_client_not_found(self, mock_collection, mock_client_init):
```

Ruff

Missing return type annotation for public function `test_get_invoice_client_not_found`

Add return type annotation: `None`

(ANN201)